sdk/core/Azure.Core/README.md

```diff
@@ -27,6 +27,8 @@ you can find the NuGet package [here](https://www.nuget.org/packages/Azure.Core)
 
 ## Usage Scenarios and Samples
 
+**❗ NOTE:** Samples in this file only apply to `Azure.*` packages. 
+
```

Comment:
Maybe it would be clearer to say "Samples in this file apply only to packages that follow Azure SDK Design Guidelines (https://azure.github.io/azure-sdk/dotnet_introduction.html). Names of such packages usually start with "Azure.*".